qemu/tests/cfg/slof_boot.cfg

```diff
@@ -0,0 +1,12 @@
+- slof_boot: install setup image_copy unattended_install.cdrom
+    type = boot
+    restart_vm = yes
+    kill_vm_on_error = yes
+    login_timeout = 240
+    variants:
+        - with_sec_size_4096_512:
+            physical_block_size=4096
```

Comment:
OK, the code likes good to me, and passed by testing with correct command line,
but pls keep the coding style, the line miss some space char, "physical_block_size = 4096" is better.